### ISSUE TITLE
{Linked:Bug10279128} [Adaptive Cards > Designer]:  In 400% zoom mode, link (Setup my work schedule from outlook on the web) present in the dialog is getting truncated.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8369]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build 25314.1010)
Browser Version: Version 112.0.1722.7 (Official build) dev (64-bit)

**Pre-Requisites:**
1. From the Windows display settings change the resolution to 1280*1024 with 100% scale.
2. Enable the Zoom to 400%.

**Repro Steps:**
1. Open the above URL in edge dev browser.
2. Navigate to the 'Card Payload Editor' and press enter key to activate it.
3. Paste the attached JSON file below.
4. Navigate to Preview Mode and press enter key to activate it.
5. Now, navigate to the 'Setup my work...' link and observe the issue.

**Actual Result:**
In 400% zoom mode, link (Setup my work schedule from outlook on the web) present in the dialog is getting truncated.

**Expected Result:** 
In 400% zoom mode, link (Setup my work schedule from outlook on the web) present in the dialog should not get truncated and everything should be clearly visible.

**User Impact:**
Sighted users who need to enlarge text and read the content are getting impacted as in 400% zoom mode, link present in the dialog is getting truncated.

**WCAG Reference:**
https://www.w3.org/TR/WCAG21/#reflow

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![MAS1 4 10- In 400% zoom mode, link (Setup my work schedule from outlook on the web) present in the dialog is getting truncated](https://user-images.githubusercontent.com/95913748/225874495-0e30760a-6032-47ff-a0a4-b1c4dccae1bd.png)

### Card JSON

```JSON
{
"schema": "http=//adaptivecards.io/schemas/adaptive-card.json",
"type": "AdaptiveCard",
"version": "1.5",
"body": [
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "Container",
"bleed": false,
"items": [
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "none",
"width": "stretch",
"type": "Container",
"bleed": true,
"items": [],
"verticalContentAlignment": "center",
"rtl": false,
"backgroundImage": "https://localhost:4321/node_modules/myhub-viva-library/dist/HeaderLine_77d65ccd075fa6644bc29b64bd426343.png"
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "none",
"type": "ColumnSet",
"bleed": false,
"columns": [
{
"type": "Column",
"separator": false,
"spacing": "none",
"verticalContentAlignment": "center",
"width": "stretch",
"isVisible": true,
"rtl": false,
"items": [
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "none",
"type": "TextBlock",
"style": "heading",
"wrap": true,
"text": "Set up or update work schedule"
}
]
}
]
}
],
"verticalContentAlignment": "center",
"rtl": false,
"headerType": "basic"
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "TextBlock",
"style": "default",
"wrap": true,
"text": "Change your work schedule for today by selecting work location.",
"size": "small",
"color": "default",
"weight": "default"
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "Container",
"bleed": false,
"items": [
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "Container",
"bleed": false,
"items": [
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "TextBlock",
"style": "default",
"wrap": false,
"text": "09:00 AM - 05:00 PM"
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "Container",
"bleed": false,
"items": [
{
"id": "LocationOptions_0",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "Input.ChoiceSet",
"isRequired": false,
"isMultiSelect": false,
"style": "compact",
"wrap": true,
"choices": [
{
"title": "In the office",
"value": "In the office"
},
{
"title": "Working remotely",
"value": "Working remotely"
}
],
"value": "Work schedule not set up",
"label": "Work location"
}
],
"verticalContentAlignment": "center",
"rtl": false
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "Medium",
"type": "TextBlock",
"style": "default",
"wrap": false,
"text": " "
}
],
"verticalContentAlignment": "center",
"rtl": false
}
],
"verticalContentAlignment": "center",
"rtl": false
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "default",
"type": "ActionSet",
"actions": [
{
"style": "positive",
"mode": "primary",
"isEnabled": true,
"id": "UpdateSchedule",
"title": "Update",
"tooltip": "Update",
"type": "Action.Submit",
"associatedInputs": "auto",
"data": {
"id": "UpdateSchedule"
}
}
]
},
{
"id": "",
"isVisible": true,
"separator": false,
"spacing": "extraLarge",
"type": "TextBlock",
"style": "default",
"wrap": true,
"text": "[Set up my work schedule from Outlook on the web](https://outlook.office.com/mail/options/calendar/workSchedule/)",
"horizontalAlignment": "center",
"size": "default"
}
],
"actions": [],
"refresh": null
}
```

**Comment by vagpt** [comment]
This issue is working fine on the environment below. hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 22H2(OS Build25330.1000)
Browser Version: 114.0.1807.1

**Snippet:**

![image](https://user-images.githubusercontent.com/95913748/235099140-5326a25c-7b91-4f44-80eb-a5c66407687e.png)
